Dart/src/com/jetbrains/lang/dart/ide/generation/CreateEqualsAndHashcodeFix.java

```diff
@@ -32,6 +35,9 @@ public CreateEqualsAndHashcodeFix(@NotNull final DartClass dartClass) {
   public void beforeInvoke(@NotNull Project project, Editor editor, PsiElement file) {
     super.beforeInvoke(project, editor, file);
     mySuperclassOverridesEqualEqualAndHashCode = doesSuperclassOverrideEqualEqualAndHashCode(myDartClass);
+
+    final DartSdk sdk = DartSdk.getDartSdk(project);
+    supportsObjectHashMethods = sdk != null && StringUtil.compareVersionNumbers(sdk.getVersion(), "2.14") >= 0;
```

Comment:
I think it's safe to assume a fresh Dart SDK by default so I changed it to 
```
sdk == null || StringUtil.compareVersionNumbers(sdk.getVersion(), "2.14") >= 0
```
It also simplifies getting DartGenerateActionTest to pass, sdk is null in it